asapdiscovery-data/asapdiscovery/data/schema_v2/ligand.py

```diff
@@ -360,13 +385,72 @@ def set_expansion(
             parent=parent, provenance=provenance
         )
 
-    def has_stereo(self) -> bool:
+    @property
+    def flattened(self) -> "Ligand":
```

Comment:
Can you add a docstring here? Not exactly sure what flattened means in this context.

Reply:
ah yep, sorry i will add